campaign_info.toml

```diff
@@ -38,4 +46,17 @@ pagename = "B20_WMDE_Test_Mobile"
 
 [mobile.test_matrix]
 device = ["iphone_xs_max", "iphone_5s", "samsung_s6", "samsung_s9_plus"]
-orientation = ["portrait"]
\ No newline at end of file
+orientation = ["PORTRAIT", "LANDSCAPE"]
```

Comment:
could you make the orientation less SCREAMING and do the uppercasing in the capability factory?